front/src/applications/operationalStudies/views/ManageTrainSchedule.tsx

```diff
@@ -52,40 +50,84 @@ export default function ManageTrainSchedule({ setDisplayTrainScheduleManagement
       }),
     }
   );
+  const tabRollingStock = {
+    title: (
+      <>
+        <img src={rollingStockPic} alt="rolling stock" height={24} />
+        <span className="ml-2">{t('tabs.rollingStock')}</span>
+      </>
+    ),
+    withWarning: rollingStockID === undefined,
```

Comment:
(Suggestion)
```suggestion
    withWarning: !rollingStockID,
```

Reply:
Risk of problem if id === 0 so I'l left this as this ;)